clab/config/template.go

```diff
@@ -68,9 +68,12 @@ func RenderAll(allnodes map[string]*NodeConfig) error {
 		log.Infof("No template names specified (-l) using: %s", strings.Join(TemplateNames, ", "))
 	}
 
-	tmpl := template.New("").
-		Funcs(jT.Funcs).
-		Funcs(gomplate.CreateFuncs(context.Background(), new(data.Data)))
+	// gomplate overrides the built-in *slice* function. Rename it to *list*
+	gfuncs := gomplate.CreateFuncs(context.Background(), new(data.Data))
+	gfuncs["list"] = gfuncs["slice"]
```

Comment:
If we delete the slice key from gfuncs, I suppose we still can use the fqdn version coll.Slice? If yes, then I wouldn't even bother with making a list alias, as I am advocating for using fully scoped names

Reply:
:+1: since the fully scoped name is still available, I'll go for the delete only